jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/bsm/bsmDepositorToOde/BsmDepositorToOde.java

```diff
@@ -0,0 +1,82 @@
+package us.dot.its.jpo.ode.bsm.bsmDepositorToOde;
```

Comment:
Just a thought - since having these inside of the application means that we have to either have to call them in our application code or run them as a unit test, neither of which can be done cleanly during runtime. It might be good to move these out to a separate directory altogether. Should be easy since they have no ODE dependencies.